example/handwritten-src/swift/TXSTextboxListenerDebugableImpl.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TXSTextboxListenerImpl.swift
+//  TextSort
+//
+//  Created by Bruno Coelho on 10/03/2017.
+//  Copyright © 2017 Dropbox, Inc. All rights reserved.
+//
+
+import UIKit
+
+@objc class TXSTextboxListenerDebugableImpl : NSObject, TXSTextboxListener {
```

Comment:
The `@objc` isn't needed when you subclass an Objective-C class (like `NSObject`). Could remove.

Would also suggest adding `final`